site/content/docs/0.0/utilities/borders.md

```diff
@@ -8,4 +8,311 @@ aliases:
+
+{{< example class="bd-example-border-utils" >}}
+<span class="border"></span>
+<span class="border-top"></span>
+<span class="border-end"></span>
+<span class="border-bottom"></span>
+<span class="border-start"></span>
+{{< /example >}}
+
+### Subtractive
+
+Or remove borders:
+
+{{< example class="bd-example-border-utils" >}}
+<span class="border border-none"></span>
+<span class="border border-top-none"></span>
+<span class="border border-end-none"></span>
+<span class="border border-bottom-none"></span>
+<span class="border border-start-none"></span>
+{{< /example >}}
+
+{{< bootstrap-compatibility >}}
+
+{{< example class="bd-example-border-utils" >}}
+<span class="border border-0"></span>
+<span class="border border-top-0"></span>
+<span class="border border-end-0"></span>
+<span class="border border-bottom-0"></span>
+<span class="border border-start-0"></span>
+{{< /example >}}
+
+{{< /bootstrap-compatibility >}}
+
+<!-- ## Color
+
+{{< callout info >}}
+Border utilities like `.border-*` that generated from our original `$theme-colors` Sass map don't yet respond to color modes, however, any `.border-*-subtle` utility will. This will be resolved in v6.
+{{< /callout >}}
+
+Change the border color using utilities built on our theme colors.
+
+{{< example class="bd-example-border-utils" >}}
+{{< border.inline >}}
+{{- range (index $.Site.Data "theme-colors") }}
+<span class="border border-{{ .name }}"></span>
+{{- end -}}
+{{< /border.inline >}}
+<span class="border border-black"></span>
+<span class="border border-white"></span>
+{{< /example >}}
+
+{{< callout info >}}
+For each `.border-*` there is a matching `.border-*-subtle` utility. In Boosted, they have exactly the same value so we decided not to display them in the example above so that you don't hesitate on which class to use.
+
+Here is a list of these extra classes:
+{{< border-subtle.inline >}}
+{{- range (index $.Site.Data "theme-colors") }}
+- `.border-{{ .name }}-subtle`
+{{- end -}}
+{{< /border-subtle.inline >}}
+{{< /callout >}}
+
+Or modify the default `border-color` of a component:
+
+<details>
+<summary>See Bootstrap examples that are incompatible with Orange Design System.</summary>
+<br>
+{{< design-callout-alert >}}
+These border color variants are just **examples illustrating the use of this color utility**. They should not be used because they do not respect the Orange Design System specifications.
+{{< /design-callout-alert >}}
+
+{{< example >}}
+<div class="mb-4">
+  <label for="exampleFormControlInput1" class="form-label">Email address</label>
+  <input type="email" class="form-control border-success" id="exampleFormControlInput1" placeholder="name@example.com">
+</div>
+
+<div class="h4 pb-2 mb-4 text-danger border-bottom border-danger">
+  Dangerous heading
+</div>
+
+<div class="p-3 bg-info bg-opacity-10 border border-info border-start-0">
+  Changing border color and width
+</div>
+{{< /example >}}
+</details>-->
+
+## Opacity
+
+{{< callout danger >}}
```

Comment:
Just a quick question on this, do we really need this callout since the remaining is hidden by a details/summary ?